docs/alternate_domains.rst

```diff
@@ -1,12 +1,18 @@
 Alternate Domains
```

Comment:
We should rename this doc entirely. Nomenclature for this feature should be "custom domains". We use "CNAME" a lot, which is not clear, and I think this is the only place we refer to this as "alternate domains".

Reply:
This will require a file rename and a redirection added.